syntax = "proto3";
option java_package = "com.alipay.rpc.common.service.facade.pb.request";
option java_multiple_files = true;
option go_package = "com.alipay.rpc.common.service.facade.pb";

message SampleServicePbRequest {
    string name = 1;
}

